src/app/donation-start/donation-start-match-confirm-dialog.component.ts

```diff
@@ -6,5 +6,10 @@ import { MAT_DIALOG_DATA } from '@angular/material/dialog';
   templateUrl: 'donation-start-match-confirm-dialog.html',
 })
 export class DonationStartMatchConfirmDialogComponent {
-  constructor(@Inject(MAT_DIALOG_DATA) public data: { cancelCopy: string, status: string, statusDetail: string, title: string }) {}
+  constructor(@Inject(MAT_DIALOG_DATA) public data: {
+    cancelCopy: string,
+    status: string,
+    statusDetail: string,
+    title: string,
+    surplusDonationInfo: string }) {}
```

Comment:
@tbg-jasonfung  minor thing but I think this is more readable if closing braces go on their own line, when params are one per line